fendermint/app/src/options/key.rs

```diff
@@ -21,6 +23,19 @@ pub struct KeyArgs {
     pub command: KeyCommands,
 }
 
+#[derive(Args, Debug)]
+pub struct AddPeer {
+    /// The path to node key file.
+    #[arg(long, short)]
+    pub node_key_file: PathBuf,
+    /// The path to a local file with persistent peers.
+    #[arg(long, short)]
+    pub local_peers_file: PathBuf,
+    /// The path to a local file with persistent peers.
+    #[arg(long, short)]
```

Comment:
You will have to say what the `short` should be because both `node_key_file` and `network_addr` start with `n`. See for example [here](https://github.com/consensus-shipyard/fendermint/blob/60ac4f47331f5c95a38b00e2d0aecace0a903caf/fendermint/app/src/options/rpc.rs#L153-L158)

Reply:
fixed